projects/go-lib/src/lib/components/go-radio/go-radio-button.component.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../../styles/variables';
+
+input[type='radio'] {
+  display: none;
+}
+
+.go-radio__custom-radio {
+  display: inline-block;
+  width: 13px;
+  height: 13px;
```

Comment:
We use `rems` all throughout, would you mind converting these to `.75rem`?

Reply:
updated